helm/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: eks-chart
 description: A Helm chart for the ACK service controller for Amazon Elastic Kubernetes Service (EKS)
-version: 1.0.5
-appVersion: 1.0.5
+version: 0.0.0-non-release-version
+appVersion: 0.0.0-non-release-version
```

Comment:
A `git fetch upstream main --tags` should will set the right tags here

Reply:
Done